learn/getting_started/customizing_relevancy.md

```diff
@@ -132,4 +132,21 @@ This will set `winnie` and `piglet` as synonyms; searching for either word will
 
 You can read more about it in our [dedicated guide](/learn/configuration/synonyms.md).
 
-The next chapter tackles more advanced topics including security and data backup.
+## Typo tolerance
+
+Meilisearch is typo tolerant by default. It will help you find relevant search results even if you make spelling mistakes or typos, e.g., searching for `swaj` instead of `swan`. The `typoTolerance` object allows you to:
+
+- Enable or disable the typo tolerance feature
+- Configure the minimum word size for typos
+- Disable typos on specific words
+- Disable typos on specific attributes
+
+Meilisearch accepts one typo for query terms containing `5` or more characters by default. If you search the movies index for `swaj`, you will not get any results.
+
+<CodeSamples id= "getting_started_typo_tolerance" />
+
+The above code sample sets the minimum word size for one typo to `4` characters. If you search for `swaj` now, you should get 298 results.
```

Comment:
Could you change the second sentence? The number of results for `swaj` is likely to change as the movies dataset is updated, so it would be better to use something more vague ("you should get some results") or something that won't change ("you should get results containing the word `swan`")